dicomImport/matRad_importDicomSteering.m

```diff
@@ -0,0 +1,286 @@
+    for j = 1:stf(i).numOfRays
+        bixelWidth_help(j,1) = stf(i).ray(j).rayPos_bev(1);
+        bixelWidth_help(j,2) = stf(i).ray(j).rayPos_bev(3);        
+    end
+    bixelWidth_help1 = unique(bixelWidth_help(:,1),'sorted');
+    bixelWidth_help2 = unique(bixelWidth_help(:,2),'sorted');
+    bixelWidth(1) = bixelWidth_help1(size(bixelWidth_help1,1)) - ...
+        bixelWidth_help1(size(bixelWidth_help1,1)-1);
+    bixelWidth(2) = bixelWidth_help2(size(bixelWidth_help2,1)) - ...
+        bixelWidth_help2(size(bixelWidth_help2,1)-1);
+    if bixelWidth(1) ~= bixelWidth(2)
+        warning('Bixelwidth differ in the different directions');
+    end
+    stf(i).bixelWidth = max(bixelWidth);
+    if isfield(pln,'bixelWidth') && ~isnan(pln.bixelWidth)
+        if stf(i).bixelWidth ~= pln.bixelWidth
+            error('multiple non-equal bixelWidths, not yet supported')
+        end
+    end
+    pln.bixelWidth = stf(i).bixelWidth;
+    % compute coordinates in lps coordinate system, i.e. rotate beam
+    % geometry around fixed patient
+    
+    % Rotation around Z axis (gantry)
+    rotMx_XY_rotated = [ cosd(pln.gantryAngles(i)) sind(pln.gantryAngles(i)) 0;
+        -sind(pln.gantryAngles(i)) cosd(pln.gantryAngles(i)) 0;
+        0                         0 1];
+    
+    % Rotation around Y axis (couch)
+    rotMx_XZ_rotated = [ cosd(pln.couchAngles(i)) 0 -sind(pln.couchAngles(i));
+        0 1                        0;
+        sind(pln.couchAngles(i)) 0 cosd(pln.couchAngles(i))];
+    
+    % Rotated Source point, first needs to be rotated around gantry, and then
+    % couch.
+    stf(i).sourcePoint =  stf(i).sourcePoint_bev*rotMx_XY_rotated*rotMx_XZ_rotated;
+    
+    % Save ray and target position in lps system.
+    for j = 1:stf(i).numOfRays
+        stf(i).ray(j).rayPos      = stf(i).ray(j).rayPos_bev*rotMx_XY_rotated*rotMx_XZ_rotated;
+        stf(i).ray(j).targetPoint = stf(i).ray(j).targetPoint_bev*rotMx_XY_rotated*rotMx_XZ_rotated;   
+        stf(i).ray(j).SSD         = NaN;
+    end
+    
+    % SSD
+    DensityThresholdSSD = 0.05;
+    for j = 1:stf(i).numOfRays
+        [alpha,~,rho,~,~] = matRad_siddonRayTracer(stf(i).isoCenter, ...
+                             ct.resolution, ...
+                             stf(i).sourcePoint, ...
+                             stf(i).ray(j).targetPoint, ...
+                             {ct.cube});
+
+            ixSSD = find(rho{1} > DensityThresholdSSD,1,'first');
+
+            if isempty(ixSSD)== 1
+                warning('Surface for SSD calculation starts directly in first voxel of CT\n');
+            end
+            
+            % calculate SSD
+            stf(i).ray(j).SSD = double(2 * stf(i).SAD * alpha(ixSSD));
+            % book keeping & calculate focus index
+            stf(i).numOfBixelsPerRay(j) = numel([stf(i).ray(j).energy]);
+            currentMinimumFWHM = interp1(machine.meta.LUT_bxWidthminFWHM(1,:),machine.meta.LUT_bxWidthminFWHM(2,:),pln.bixelWidth);
```

Comment:
What are you doing here? Why do we need the bixeWidth at this point? This looks fishy.